momento-sdk/src/main/java/momento/sdk/messages/CacheDictionaryFetchResponse.java

```diff
@@ -37,20 +37,20 @@ public Map<String, String> valueDictionaryStringString() {
     }
 
     /**
-     * Gets the retrieved values as a dictionary of UTF-8 string keys and values.
+     * Gets the retrieved elements as a map of UTF-8 string fields to UTF-8 string values.
      *
-     * @return the dictionary.
+     * @return The map.
      */
-    public Map<String, String> valueDictionary() {
-      return valueDictionaryStringString();
+    public Map<String, String> valueMap() {
+      return valueMapStringString();
     }
 
     /**
-     * Gets the retrieved value as a dictionary of UTF-8 String keys and byte array values
+     * Gets the retrieved elements as a map of UTF-8 string fields to byte array values.
      *
-     * @return the dictionary.
+     * @return The map.
      */
-    public Map<String, byte[]> valueDictionaryStringBytes() {
+    public Map<String, byte[]> valueMapStringBytes() {
```

Comment:
"Bytes" or "ByteArray"? I am looking for consistency across `Set` and `SortedSet`. I believe we want `ByteArray` but let me know if a decision was made for these compound names.

Reply:
ByteArray makes more sense; I'll change it. I imagine Bytes was used because valueDictionaryByteArrayByteArray is unpleasantly long.